classes/mender-install.bbclass

```diff
@@ -13,11 +6,4 @@ VIRTUAL-RUNTIME_init_manager = "systemd"
 DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
 VIRTUAL-RUNTIME_initscripts = ""
 
-
-#Make sure we are creating sdimg with all needed partitioning.
-IMAGE_CLASSES += "sdimg"
-IMAGE_FSTYPES += "sdimg"
-
-IMAGE_INSTALL_append = " u-boot-fw-utils mender ca-certificates"
-
-
+IMAGE_INSTALL_append = " mender ca-certificates"
```

Comment:
I suspect the ca-certificates was in order for S3 links to work.

`packagegroup-core-boot` sounds like a good idea, but this is a pretty big task, not really related to what I'm doing in this PR, so I won't fix it here. But it's something to consider for later.